src/messages/MOSDOp.h

```diff
@@ -242,14 +236,14 @@ struct ceph_osd_request_head {
       ::encode_nohead(snaps, payload);
     } else {
       header.version = HEAD_VERSION;
-      ::encode(client_inc, payload);
+      ::encode(pgid, payload);
```

Comment:
Features could somehow determine to which version of ceph-osd we are sending the message?